docker_digitaltwin/api_data/insert_apis_to_db.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Sep  6 10:49:19 2021
+
+@author: pkh35
```

Comment:
Could you add documentation strings to each python file and the the complex functions, it would make it easy to summarise what each file does for a reader